ui/app/models/database/credential.js

```diff
@@ -0,0 +1,11 @@
+import Model, { attr } from '@ember-data/model';
+
+export default Model.extend({
```

Comment:
I think this model is unused now, right?

Reply:
Nope, confirmed this is being used.